satellite/admin.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellite
+
+import (
+	"bytes"
+	"context"
+	"encoding/hex"
+
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/pkg/macaroon"
+	"storj.io/storj/satellite/console"
+)
+
+// AdminConfig represents configuration for admin-bypass-type operations
+type AdminConfig struct {
+	AdminHead   string `help:"the macaroon head to expect for admin keys. In hex" default:""`
+	AdminSecret string `help:"the macaroon secret to expect for admin keys. In hex" default:""`
```

Comment:
The config is already called `Admin` so `Head` and `Secret` would be sufficient.